revdep/problems.md

```diff
@@ -0,0 +1,17881 @@
+
+</details>
+
+## Newly broken
+
+*   checking examples ... ERROR
+    ```
+    Running examples in ‘autoplotly-Ex.R’ failed
+    The error most likely occurred in:
+    
+    > ### Name: autoplotly
+    > ### Title: Automatic Visualization of Popular Statistical Results Using
+    > ###   'plotly.js' and 'ggplot2'
+    > ### Aliases: autoplotly
+    > 
+    > ### ** Examples
+    > 
+    ...
+    > p <- autoplotly(prcomp(iris[c(1, 2, 3, 4)]), data = iris,
+    +                 colour = 'Species', label = TRUE, label.size = 3, frame = TRUE)
+    Warning: `aes_string()` was deprecated in ggplot2 3.0.0.
+    ℹ Please use tidy evaluation idioms with `aes()`.
+    ℹ See also `vignette("ggplot2-in-packages")` for more information.
+    ℹ The deprecated feature was likely used in the ggfortify package.
+      Please report the issue at <https://github.com/sinhrks/ggfortify/issues>.
+    Error in pm[[2]] : subscript out of bounds
+    Calls: autoplotly ... autoplotly.default -> ggplotly -> ggplotly.ggplot -> gg2list
+    Execution halted
+    ```
+
+*   checking tests ... ERROR
+    ```
+      Running ‘testthat.R’
+    Running the tests in ‘tests/testthat.R’ failed.
+    Complete output:
+      > library(testthat)
+      > library(autoplotly)
+      > 
+      > test_check("autoplotly")
+      [ FAIL 3 | WARN 1 | SKIP 0 | PASS 1 ]
+      
+      ══ Failed tests ════════════════════════════════════════════════════════════════
+    ...
+          ▆
+       1. ├─autoplotly::autoplotly(...) at test_all.R:26:3
+       2. └─autoplotly:::autoplotly.default(...)
+       3.   ├─plotly::ggplotly(...)
+       4.   └─plotly:::ggplotly.ggplot(...)
+       5.     └─plotly::gg2list(...)
+      
+      [ FAIL 3 | WARN 1 | SKIP 0 | PASS 1 ]
+      Error: Test failures
+      Execution halted
+    ```
+
+# bakeoff
+
+<details>
+
+* Version: 0.2.0
+* GitHub: https://github.com/apreshill/bakeoff
+* Source code: https://github.com/cran/bakeoff
+* Date/Publication: 2022-10-21 15:25:10 UTC
+* Number of recursive dependencies: 31
+
+Run `revdepcheck::cloud_details(, "bakeoff")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking examples ... ERROR
+    ```
+    Running examples in ‘bakeoff-Ex.R’ failed
+    The error most likely occurred in:
+    
+    > ### Name: bakeoff_palette
+    > ### Title: A *bakeoff* palette generator
+    > ### Aliases: bakeoff_palette
+    > 
+    > ### ** Examples
+    > 
+    > bakeoff_palette("showstopper")
+    ...
+     18. └─vctrs (local) `<fn>`()
+     19.   └─vctrs::vec_default_ptype2(...)
+     20.     ├─base::withRestarts(...)
+     21.     │ └─base (local) withOneRestart(expr, restarts[[1L]])
+     22.     │   └─base (local) doWithOneRestart(return(expr), restart)
+     23.     └─vctrs::stop_incompatible_type(...)
+     24.       └─vctrs:::stop_incompatible(...)
+     25.         └─vctrs:::stop_vctrs(...)
+     26.           └─rlang::abort(message, class = c(class, "vctrs_error"), ..., call = call)
```

Comment:
This is also a recurring error, it think it might be because vctrs is too strict. Or at least: people use the lack of strictness we had earlier.